lib/src/resolver.dart

```diff
@@ -157,7 +157,9 @@ class Loader {
   /// Returns `null` if the resource could not be loaded.
   Future<List<String>> load(String path) async {
     try {
-      return File(path).readAsLines();
+      final lines = await File(path).readAsLines();
+      // Ensure `readAsLines` runs within the try block so errors are caught.
```

Comment:
I don't understand this comment

`return await File(path).readAsLines();` should be sufficient and clear. In a future SDK release the `await` won't be necessary.

Reply:
ack!